### ISSUE TITLE
Problem updating the root domain on Porkbun

### ISSUE THREAD

**Issue by ShadowDancerCo** [ddclient/ddclient#707]
Hello, I'm using ddclient v3.11.2 on a Ubuntu Server machine and not sure if it's an issue with my config file, but ddclient doesn't seem to update all the records correctly.

For example, the `*.example.com` record would be updated correctly, but not the `example.com` record.

I tried to find examples for Porkbun configuration, but everything I tried doesn't work.

Here's 3 examples of what I tried and each time, the `*.example.com` is fine, but not `example.com`:
```
protocol=porkbun
apikey=
secretapikey=
example.com,*.example.com


protocol=porkbun
apikey=
secretapikey=
on-root-domain=yes example.com,*.example.com


protocol=porkbun
apikey=
secretapikey=
root-domain=example.com
*.example.com
```

Any help would be very appreciated. Thanks!

**Comment by rhansen** [comment]
I believe this was fixed by #627.  Please try a recent build of `master`:  https://launchpad.net/~ddclient/+archive/ubuntu/daily

**Comment by ShadowDancerCo** [comment]
I tried a recent build and indeed it seems to work fine, thanks!

Though it didn't worked at first try, I had to do adjustments to my config file afterwards since I had an "Invalid domain" error for `*.example.com`.

For future reference, here's what I put for it to work:
```
protocol=porkbun
apikey=
secretapikey=
root-domain=example.com
example.com,*.example.com
```
